struct vs class warnings emitted during clang build

Various structs appear to be forward-declared as classes elsewhere, and that is a source of warnings.

> e.g. In file included from /home/travis/build/KazDragon/terminalpp/src/detail/element_difference.cpp:3:
> /home/travis/build/KazDragon/terminalpp/include/terminalpp/behaviour.hpp:9:1: warning: 'behaviour' defined as a struct here but previously declared as a class [-Wmismatched-tags]
struct behaviour
> ^
> /home/travis/build/KazDragon/terminalpp/include/terminalpp/detail/element_difference.hpp:8:1: note: did you mean struct here?
> class behaviour;

--- a/include/terminalpp/detail/element_difference.hpp
+++ b/include/terminalpp/detail/element_difference.hpp
@@ -5,8 +5,8 @@
 
 namespace terminalpp {
 
-class behaviour;
-class element;
+struct behaviour;
+struct element;
 
 namespace detail {
 

--- a/include/terminalpp/detail/terminal_cursor_control.hpp
+++ b/include/terminalpp/detail/terminal_cursor_control.hpp
@@ -4,8 +4,8 @@
 
 namespace terminalpp {
 
-class behaviour;
-class point;
+struct behaviour;
+struct point;
 class terminal;
 
 namespace detail {

